server/test/resolve.cpp

```diff
@@ -1926,7 +1983,32 @@ TEST_CASE("Resolve by Tags")
             auto g    = groupLink.create();
             auto info = g.resolve();
             REQUIRE(info.size() == 1);
-            CHECK(info[0].name == "srv11");
+            CHECK(info[0].name == "srv1");
+            g.remove();
+        }
+
+        // And operator | IsNot
+        {
+            std::string group = R"(
+                name  : ByTag
+                rules :
+                    operator  : AND
+                    conditions:
+                      - field    : tags
+                        operator : ISNOT
+                        value    : black
+            )";
+
+            Group groupLink;
+
+            if (auto ret = pack::yaml::deserialize(group, groupLink); !ret) {
```

Comment:
is it a more direct way to create/initialize groupLink without any deserializer?
If yes, it shall be very usefull to add this in UT, as code example...